### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fiver-watson**
Align README with recent changes in other Enduro repositories

### CODE REVIEW

**Review by jraddaoui — COMMENTED** [review]
Should we also modify the docs index?

https://github.com/artefactual-sdps/enduro/blob/main/docs/src/index.md

**Comment by fiver-watson** [comment]
ok @jraddaoui, i added another commit for the docs index

File: docs/src/index.md [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
-
-This version of Enduro uses [a3m](https://github.com/artefactual-labs/a3m)
-instead of Archivematica to preserve digital objects, alongside
-[MinIO](https://min.io/) for object storage and [Temporal](https://temporal.io/)
-to manage the workflow. This combination of tools is intended to be lightweight,
-scalable, and easy to install.
+[Enduro][Enduro] is a new application under development by Artefactual Systems.
+Originally created as a more stable replacement for Archivematica’s
+[automation-tools][automation-tools] library of scripts, it has since evolved
+into a f lexible tool to be paired with preservation applications to provide
```
Comment (jraddaoui) [inline docs/src/index.md]: f lexible

**Review by jraddaoui — CHANGES_REQUESTED** [review]
Looks great! Just a typo ;)

File: docs/src/index.md (hunk 1 above)
Comment (fiver-watson) [reply docs/src/index.md]: :sob:

Comment (fiver-watson) [reply docs/src/index.md]: ok, just committed a fix

**Review by jraddaoui — APPROVED** [review]
Thanks!

**Pull request merged**